soccer/gameplay/tactics/penalty.py

```diff
@@ -59,7 +59,7 @@ def execute_setup(self):
         self.robot.face(main.ball().pos)
 
     def on_enter_ready(self):
-        kick = skills.pivot_kick.PivotKick()
+        kick = skills.line_kick.LineKick()
```

Comment:
Should this be kept in or reverted to pivot?

Reply:
IIRC we actually tested this on the field and line kick performed better so we should leave it

Reply:
Fair. Is this a thing that we will want to revert when we have better motion control? If so can we mark it with a TODO?

Reply:
It's a stylistic change in my mind. I think it's fine

Reply:
ACK.